packages/frontend/editor-ui/src/app/components/MainHeader/WorkflowPublishModal.vue

```diff
@@ -53,7 +53,15 @@ const wfHasAnyChanges = computed(() => {
 
 const hasNodeIssues = computed(() => workflowsStore.nodesIssuesExist);
 
-const nodesWithIssuesCount = computed(() => workflowsStore.nodesWithIssuesCount);
+const nodesWithIssues = computed(() =>
+	workflowsStore.workflow.nodes.filter((node) => {
```

Comment:
We already have this filtering in the `workflows.store.ts` in `nodesIssuesExist` let's just extract it into a getter in the store called something nodesWithIssues and use it here and in the `nodesIssuesExist`